src/recensio/plone/browser/templates/review.pt

```diff
@@ -201,18 +201,20 @@
                                  "
                                  condition="data/value"
                         >
-                          <dt>
+                          <dt class="${field_name}__label">
```

Comment:
I do not see the need of adding this `__label`  suffix. What is it for?

Reply:
Fancy stuff!

Does it make sense to switch to a bootstrap accordion structure, see https://getbootstrap.com/docs/5.0/components/accordion/?
Then we could have:
`<div class="accordion-item ${field_name}">`.
Do not waste much time on that.
Just in case, remove the `__$suffix` thing. I am not buying this BEM thing right now

Reply:
There is another reason: the generated CSS classes might be too generic and get some unwanted styles pulled in.
Without the `__label` or `__value` suffix, we end up with classes like `title`, `subjects`, `editor`.
The BEM naming system is actually good in avoiding ambiguous names.

Given that thread I'd keep the `__label` and `__value` suffixes or remove that commit from this PR.

Reply:
Can you show me a CSS rule that requires this classes?

Reply:
You mean the `${field_name}__label` ?‌ there is no such rule yet, it's only adding semantic context.

Initially that was only for me debugging the metadata column to see which output belongs to which field and as I‌ though that to be useful I‌ kept it in a separate commit.

I'm removing that commit so that we can go forward with the original scrum-1191 ticket